### PR TITLE
Query per guid

### PR DESCRIPTION
**Pull request by ruimangas**

### CODE REVIEW

**Comment by nunofmn** [comment]
LGTM. Maybe @BernardoMG could write a test for the search by GUID endpoint.

**Comment by BernardoMG** [comment]
@nunofmn sure! I've already told to @ruimangas that I will write the tests for this new feature until Monday afternoon

File: server/specs/hyperties_spec.rb [hunk 1]
```diff
@@ -159,6 +159,70 @@
       end
     end
 
+    describe 'get hyperties by guid' do
+      Airborne.configure do |config|
+        config.base_url = ENV["HOST"].dup << '/hyperty/guid'
+      end
+    end
+
+      it 'should return four hiperties' do
```
Comment (nunofmn) [inline server/specs/hyperties_spec.rb]: "Hyperties" not "hiperties" 👍

**Comment by ruimangas** [comment]
@nunofmn, @BernardoMG just added some tests. I think we can merge this

File: server/specs/hyperties_spec.rb [hunk 2]
```diff
@@ -159,6 +159,70 @@
...
+          expect(json_body[:hyperty3][:guid]).to eql("guid4")
+          expect(json_body[:hyperty6][:guid]).to eql("guid4")
+        end
+      end
+
+      it 'should return one hyperty' do
+        get '/guid_10' do
+          expect_status(404)
+          expect_json_sizes(1)
+          expect_json_keys("hypert3", [:descriptor, :startingTime, :lastModified, :expires, :resources, :dataSchemes, :status, :runtime, :p2pRequester, :p2pHandler, :guid])
```
Comment (ruimangas) [inline server/specs/hyperties_spec.rb]: hyperty3 not hypert3. It should be failing some test

**Pull request merged**